src/components/screens/lockedBalance/lockedBalance/unlockingList.js

```diff
@@ -5,10 +5,10 @@ import { getDelayedAvailability, isBlockHeightReached } from '../../../../utils/
 import LiskAmount from '../../../shared/liskAmount';
 import { tokenMap } from '../../../../constants/tokens';
 
-const getPendingTime = ({ unvoteHeight, delegateAddress }, currentBlockHeight, { address }) => {
+const getPendingTime = ({ height, delegateAddress }, currentBlockHeight, { address }) => {
   const isSelfVote = address === delegateAddress;
   const delayedAvailability = getDelayedAvailability(isSelfVote);
-  const awaitingBlocks = delayedAvailability - (currentBlockHeight - unvoteHeight);
+  const awaitingBlocks = delayedAvailability - (currentBlockHeight - height.start);
```

Comment:
If `height.end` Is when the balance is able to be unlocked then we don't need `delayed availability`. We should just substract `currentBlockHeight` to `height.end` : ```currentBlockHeight - height.end```

Reply:
Thus we won't need `getDelayedAvailability` at all

Reply:
Good point. that part is already calculated by Lisk Service.